simple_history/management/commands/clean_duplicate_history.py

```diff
@@ -115,6 +119,8 @@ def _process_instance(self, instance, model, stop_date=None, dry_run=True):
             for f2 in o_qs[1:]:
                 entries_deleted += self._check_and_delete(f1, f2, dry_run)
                 f1 = f2
+                if entries_deleted > 0 and entries_deleted % batch_size == 0:
```

Comment:
Is this form of batching good enough or is it important that we break up the transaction as well?